be/src/storage/rowset/indexed_column_reader.h

```diff
@@ -142,12 +140,12 @@ class IndexedColumnReader {
     Status read_page(RandomAccessFile* read_file, const PagePointer& pp, PageHandle* handle, Slice* body,
                      PageFooterPB* footer) const;
 
+    const IndexReadOptions& _opts;
```

Comment:
Is it appropriate to use reference here? It seems that there is no guarantee the referenced object will live longer than the `IndexedColumnReader` object.